src/components/logpile.vue

```diff
@@ -139,16 +146,46 @@ export default {
           svg_add
             .selectAll(".bar")
             .data(data, function(d) { return d.Type })
-            .enter()
+          .enter()
             .append("rect")
             .classed("bar", true)
             .attr("class", d => { return "bar " + d.Type }) // to grab in interaction
             .attr("x", d => xScale(x(d)))
             .attr("y", y_height)
-            .attr("width", 2)
+            .attr("width", 10)
             .attr("height", y_height)
-            .attr("opacity", 0.85)
-            .attr("fill", "pink")    // TODO: color based on type
+            .attr("fill", "royalblue")
+
+            var file = 'salineLake.jpg';
+
+          svg_add
```

Comment:
Not a big deal, but I think you could go ahead and append this whole section onto the one above, where you're fist creating the rectangles, rather than selecting them again here.